### PR TITLE
Russian traslation updated

### PR DESCRIPTION
**Pull request by gostsdmitry**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
-"Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of "
-"%s%s%s branch %s%s%s\n"
+"Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of %s%s"
+"%s branch %s%s%s\n"
 msgstr ""
-"Информация: конец срока поддержки для закреплённого %s//%s наступил в пользу "
```
Comment (gasinvein) [inline po/ru.po:3637]: Please keep the translation consistent.
"end-of-life" was translated this way in other places, so you should either also change it everywhere, or, better, stick to the current "end-of-life" translation.

File: po/ru.po [hunk 2]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: runtime %s%s%s branch %s%s%s is end-of-life, in favor of %s%s%s branch "
 "%s%s%s\n"
 msgstr ""
 "\n"
-"Информация: %s%s %s прекращает свое существование, в пользу %s\n"
+"Информация: среда выполнения %s%s%s ветвь %s%s%s завершает жизнь, в пользу "
```
Comment (gasinvein) [inline po/ru.po:3649]: Same here, this change introduces inconsistent "end-of-life" translation.

File: po/ru.po [hunk 3]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
 msgid ""
 "\n"
-"Info: app %s%s%s branch %s%s%s is end-of-life, in favor of %s%s%s branch "
-"%s%s%s\n"
+"Info: app %s%s%s branch %s%s%s is end-of-life, in favor of %s%s%s branch %s%s"
+"%s\n"
 msgstr ""
 "\n"
-"Информация: %s%s %s прекращает свое существование, в пользу %s\n"
+"Информация: приложение %s%s%s ветвь %s%s%s завершает жизнь, в пользу %s
```
Comment (gasinvein) [inline po/ru.po:3660]: Same here, this change introduces inconsistent "end-of-life" translation.

File: po/ru.po [hunk 4]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
 #: app/flatpak-cli-transaction.c:761
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, with reason:\n"
 msgstr ""
-"Информация: конец срока поддержки закреплённого %s//%s наступил по причине:\n"
+"\n"
+"Информация: (закреплено) среда выполнения %s%s%s ветвь %s%s%s завершает жизнь по "
```
Comment (gasinvein) [inline po/ru.po:3671]: Same here, this change introduces inconsistent "end-of-life" translation.

File: po/ru.po [hunk 5]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
 #: app/flatpak-cli-transaction.c:767
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: runtime %s%s%s branch %s%s%s is end-of-life, with reason:\n"
 msgstr ""
 "\n"
-"Информация: %s%s %s прекращает свое существование, по причине:\n"
+"Информация: среда выполнения %s%s%s ветвь %s%s%s завершает жизнь по причине:\n"
```
Comment (gasinvein) [inline po/ru.po:3681]: Same here, this change introduces inconsistent "end-of-life" translation.

File: po/ru.po [hunk 6]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
 #: app/flatpak-cli-transaction.c:770
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: app %s%s%s branch %s%s%s is end-of-life, with reason:\n"
 msgstr ""
 "\n"
-"Информация: %s%s %s прекращает свое существование, по причине:\n"
+"Информация: приложение %s%s%s ветвь %s%s%s завершает жизнь по причине:\n"
```
Comment (gasinvein) [inline po/ru.po:3690]: Same here, this change introduces inconsistent "end-of-life" translation.

**Review by gasinvein — CHANGES_REQUESTED** [review]

**Comment by gostsdmitry** [comment]
Excuse me, did you block the passage of checks or am I making some mistakes somewhere in the translation file?

File: po/ru.po [hunk 7]
```diff
@@ -3539,74 +3540,73 @@ msgid "Info: %s was skipped"
...
 msgid "Error: %s%s%s not installed"
-msgstr "%s/%s/%s не установлено"
+msgstr "Ошибка: %s%s%s не установлено"
 
 #: app/flatpak-cli-transaction.c:530
-#, fuzzy, c-format
+#, c-format
 msgid "Warning: %s%s%s needs a later flatpak version"
-msgstr "%s требует более новую версию flatpak"
+msgstr "Предупреждение: %s%s%s требуется более новая версия flatpak"
```
Comment (gasinvein) [inline po/ru.po:3565]:
```suggestion
msgstr "Предупреждение: %s%s%s требует более новую версию flatpak"
```

File: po/ru.po [hunk 8]
```diff
@@ -3539,74 +3540,73 @@ msgid "Info: %s was skipped"
...
 
 #: app/flatpak-cli-transaction.c:533
-#, fuzzy, c-format
+#, c-format
 msgid "Error: %s%s%s needs a later flatpak version"
-msgstr "%s требует более новую версию flatpak"
+msgstr "Ошибка: %s%s%s требуется более новая версия flatpak"
```
Comment (gasinvein) [inline po/ru.po:3570]:
```suggestion
msgstr "Ошибка: %s%s%s требует более новую версию flatpak"
```

File: po/ru.po [hunk 9]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
 "\n"
-"Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of "
-"%s%s%s branch %s%s%s\n"
+"Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of %s%s"
+"%s branch %s%s%s\n"
 msgstr ""
-"Информация: конец срока поддержки для закреплённого %s//%s наступил в пользу "
-"%s\n"
+"\n"
+"Информация: (закреплено) среда выполнения %s%s%s ветвь %s%s%s прекращает свое существование, в "
```
Comment (gasinvein) [inline po/ru.po:3638]:
```suggestion
"Информация: (закрепленная) среда выполнения %s%s%s ветвь %s%s%s прекращает свое существование, в "
```

File: po/ru.po [hunk 10]
```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
...
 #: app/flatpak-cli-transaction.c:761
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, with reason:\n"
 msgstr ""
-"Информация: конец срока поддержки закреплённого %s//%s наступил по причине:\n"
+"\n"
+"Информация: (закреплено) среда выполнения %s%s%s ветвь %s%s%s прекращает свое существование по "
```
Comment (gasinvein) [inline po/ru.po:3671]:
```suggestion
"Информация: (закрепленная) среда выполнения %s%s%s ветвь %s%s%s прекращает свое существование по "
```

**Review by gasinvein — COMMENTED** [review]
Some nitpicks, otherwise lgtm now.

**Review by gasinvein — APPROVED** [review]

**Pull request merged**